db/seeds.rb

```diff
@@ -1,9 +1,4 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
-#     MovieGenre.find_or_create_by!(name: genre_name)
-#   end
+# db/seeds.rb
+5.times do
+  Message.create(content: Faker::Lorem.sentence)
+end
```

Comment:
- [ ] When I try to run the seed command to generate random greetings, I get the error shown on the screenshot below. Also, I ran the rails c command to manually generate random greetings, and I got the same error. Kindly fix that so greetings are generated and then the app can run and be tested as expected 👍🏽

<img width="758" alt="Screenshot 2023-12-02 at 2 29 53 PM" src="https://github.com/hetrox8/hello-rails-back-end/assets/50941074/c3db3fba-f0c0-4cbf-b6e4-e9a8473defa0">

Reply:
## changes made 
1. modift seeds.rb